src/docs/get-started/codelab.md

```diff
@@ -15,6 +15,9 @@ diff2html: true
 {% asset get-started/startup-namer-part-1 alt="The app that you'll be building" class='site-image-right' %}
 <?code-excerpt path-base="codelabs/startup_namer"?>
 
+{%- comment %}In this page, color highlights like diff additions (in green){% endcomment -%}
+<style>pre .highlight {background-color: #dfd;}</style>
```

Comment:
In most situations (in this page), the highlighted code is code that is _added_. Regardless, I'm suggesting this change in color because of the limitations of the current diff plugin, which I plan to address shortly. In that sense, this change in color is temporary.

(Cool resource, thanks.)